projects/lux/src/lib/geolocation/geolocation.component.html

```diff
@@ -0,0 +1,55 @@
+<div class="lux-geolocation">
```

Comment:
Un detallito más: necesito que el div raiz tenga `[id]="inputId"`  (como lo tiene el autocomplete) porque al emparejar el <label for="x"> quero que el x apunte al div raiz -> por accesibilidad.